Libraries/Boards/MAX78002/EvKit_V1/Include/board.h

```diff
@@ -37,6 +37,13 @@
  */
 
 #include <stdio.h>
+#ifdef TFT_ADAFRUIT
```

Comment:
@vicloginov FYI I moved these to board.h

Reply:
No problem